app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -369,7 +369,9 @@ public void onSwipeBottom() {
       } else {
         Log.e(TAG_KIWIX, "Problem finding the content, no more OnCreate() code");
         // What should we do here? exit? I'll investigate empirically.
-        return;
+        // It seems unpredictable behaviour if the code returns at this point as yesterday
```

Comment:
Comments like this don't seem like a great fit for a PR. Maybe this would be best opened as a draft pr? I think this is a work in progress

Reply:
This is the current behaviour so I think this is a ready pull request. It can be followed up on in the future.

Reply:
What I'd like to propose is to accept this 'fix' since that's what it is - it improves the external behaviour of the current codebase. And we then have a separate discussion about how to develop and support custom Android apps which is a rich topic where various people beyond the 3 of us will provide various perspectives and we can then decide how we *want* the code to behave and how much we want the custom app to be self-healing, etc.

Reply:
This doesn't seem like a fix for a bug? The app still crashes right?
I suppose the scenario is a bit unlikely and the code is no worse off really.

Saying yesterday/today sounds very nontechnical to me but that is a very minor point

Reply:
grand